app/cdap/components/GridTable/index.tsx

```diff
@@ -235,6 +237,12 @@ export default function GridTable() {
             })}
         </TableBody>
       </Table>
+      <FooterPanel
+        recipeStepsCount={0}
+        dataCounts={{ rowCount: 1000, columnCount: 20 }}
+        setOpenColumnViewHandler={setOpenColumnViewHandler}
```

Comment:
i need explanation regarding this prop, why we have 4 props
setOpenColumnViewHandler and columnViewPanelOpened seem to be doing same thing
plz have proper look and update the code